ustriage/ustriage.py

```diff
@@ -765,12 +774,22 @@ def launch():
     date_range = {'start': args.start_date,
                   'end': args.end_date}
 
+    if args.age == -1:
```

Comment:
I think we can make set the argparse default to `False`, so we don't need the special `-1` value and we don't need the
```
else:
    age = False
```
we have right below.

Reply:
I was a bit afraid of type=int and default False.
But you are right it works fine and can simplify the check even more.
If we allow the type to flip int/bool we do not need the `age` variable at all.